xonsh/tools.py

```diff
@@ -555,6 +555,21 @@ def ensure_int_or_slice(x):
         return int(x)
 
 
+def is_completions_display_value(x):
+    return x in {'none', 'single', 'multi'}
+
+
+def to_completions_display_value(x):
+    if str(x).lower() in {'none', 'false'}:
+        return 'none'
+    if str(x).lower() in {'multi', 'true'}:
+        return 'multi'
+    if str(x).lower() in {'single'}:
```

Comment:
I am still unsure if I understand how you would write the conditions. Do you mean something like this?:

```python
from warnings import warn


def to_completions_display_value(x):
    x = str(x).lower()
    if x in {'none', 'false'}:
        x = 'none'
    elif x in {'multi', 'true'}:
        x = 'multi'
    elif x in {'single'}:
        x = 'single'
    else:
        warn('"{}" is not a valid value for $COMPLETIONS_DISPLAY. '.format(x) +
             'Using "multi".')
        x = 'multi'
    return x
```

(I personally find the multiple returns more clear and concise than this, but I will use the code style that you prefer of course.)

I have a problem with `warnings.warn()`, though:
```python
to_completions_display_value('Invalid value')
```
outputs
```
test.py:15: UserWarning: "invalid value" is not a valid value for $COMPLETIONS_DISPLAY. Using "multi".
  'Using "multi".')
```
`warn()` seems to print a line of the original code there, and I don't know how to turn off this behavior without ugly hacks. Did I do anything wrong there?